source/OutfitterPanel.cpp

```diff
@@ -980,3 +980,26 @@ const vector<Ship *> OutfitterPanel::GetShipsToOutfit(bool isBuy) const
 
 	return shipsToOutfit;
 }
+
+
+
+int OutfitterPanel::FindItem(const string &text) const
+{
+	int bestIndex = 9999;
+	int bestItem = -1;
+	vector<Zone>::const_iterator it = zones.begin();
```

Comment:
We usually just use `auto` for iterators.
```suggestion
	auto it = zones.begin();
```